### DISCUSSION TITLE
Can't log in when run behind nginx reverse proxy.

### DISCUSSION THREAD

**Discussion by Rajeh16** [sct/overseerr#2804] [Help]
It just gets stuck after I type my credentials in or log in with Plex. I am not sure what the issue is, especially since it used to work before I moved to a new server. Proxy support is enabled in settings.

**Comment by batty1991** [comment]
I had a similar issue using nginx and Cloudflare. I switched the Cloudflare SSL/TLS setting from "Flexible" to "Full" and that works.
